cli.js

```diff
@@ -60,8 +55,8 @@ const scriptArgs = process.argv.slice(3)
   .join(' ')
 
 searchForScript(__dirname, scriptPath)
-  .then(
-    foundPath => executeScript(foundPath, scriptArgs),
-    triedPaths => throwSearchFail(scriptPath, triedPaths)
-  )
-  .catch(() => process.exit(1))
+  .then(foundPath => executeScript(foundPath, scriptArgs))
+  .catch(e => {
+    console.error(e.toString())
+    process.exit(1)
+  })
```

Comment:
It's not sugar though is my point, it's different :/. It's similar for 99% of use cases, but unless bluebird isn't adhering to the spec then they are actually different, and it's worth remembering. In fact, it's the very reason we right out services as `.then(success, error)` rather than `.then(success).catch(error)`.

![screen shot 2016-02-22 at 10 03 56 pm](https://cloud.githubusercontent.com/assets/6439050/13240393/32eab1ba-d9b0-11e5-8c3c-210dbffc1647.png)